### PR TITLE
Global Footer

### PR DESCRIPTION
**Pull request by benbarnett**
Styles taken from gov.uk, but adapted and converted to BEM conventions.

### CODE REVIEW

**Comment by jiannis** [comment]
:ok_hand:

File: app/views/components/_footer_categories.html.erb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  </div>
+  <div class="footer-categories__one-column">
+    <h3 class="footer-categories__heading">Illness and death</h3>
+    <ul class="footer-categories__list">
+      <li><a href="/benefits">Ill health</a></li>
+      <li><a href="/when-you-die">When you die</a></li>
+    </ul>
+  </div>
+  <div class="footer-categories__one-column">
+    <h3 class="footer-categories__heading">More...</h3>
```
Comment (andrewgarner) [inline app/views/components/_footer_categories.html.erb]:
```html
More&hellip;
```
?

Comment (benbarnett) [inline app/views/components/_footer_categories.html.erb]: Yep will do

Comment (benbarnett) [inline app/views/components/_footer_categories.html.erb]: https://github.com/guidance-guarantee-programme/pension_guidance/commit/35f883933e02f586f18228005dd8831d8408f351

**Pull request merged**